conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/rpc/BitcoinRpcClientTest.scala

```diff
@@ -61,10 +62,10 @@ class BitcoinRpcClientTest extends WordSpec with Matchers with BitcoinFixtures {
 
   trait BitcoinClientFixtures {
```

Comment:
Could you kindly provide a short description of why and how to use this trait?

Reply:
Done.